pkg/alerts/policies.go

```diff
@@ -137,3 +241,64 @@ type alertPolicyResponse struct {
 type alertPolicyRequestBody struct {
 	Policy Policy `json:"policy"`
 }
+
+type alertQueryPolicyCreateResponse struct {
+	QueryPolicy QueryPolicy `json:"alertsPolicyCreate"`
+}
+
+type alertQueryPolicyUpdateResponse struct {
+	QueryPolicy QueryPolicy `json:"alertsPolicyCreate"`
+}
+type alertQueryPolicyResponse struct {
+	Actor struct {
+		Account struct {
+			Alerts struct {
+				PoliciesSearch struct {
+					NextCursor *string        `json:"nextCursor"`
+					Policies   []*QueryPolicy `json:"policies"`
+				} `json:"policiesSearch"`
+			} `json:"alerts"`
+		} `json:"account"`
+	} `json:"actor"`
+}
+
+type alertQueryPolicyDeleteRespose struct {
+	AlertsPolicyDelete struct {
+		ID int `json:"id,string"`
+	} `json:"alertsPolicyDelete"`
+}
+
+const (
+	alertPolicyQueryPolicy = `query($accountID: Int!, $policyIDs: [ID!]!) {
+		actor {
+			account(id: $accountID) {
+				alerts {
+					policiesSearch(searchCriteria: {
+						ids: $policyIDs
+					}) {
+						policies {
+							id
+							name
+							incidentPreference
+						} } } } } }`
+
+	alertsPolicyCreatePolicy = `mutation CreatePolicy($accountID: Int!, $policy: AlertsPolicyInput!){
```

Comment:
Do we need the last `Policy` on these constants? Feels redundant :yodawg:

Reply:
Not sure what you mean here.  Are you saying we do some string embedding for the constants?

Reply:
I think he means the variable name has `Policy` twice.

Reply:
Yeah, `alertsPolicyCreatePolicy` could be `alertsPolicyCreate` (though we should also have something like @sanderblue said around gql / mutation / query.